phenotools-cli/src/main/java/org/phenopackets/phenotools/examples/NemalineMyopathyPrenatal.java

```diff
@@ -0,0 +1,214 @@
+        builder.sampledTissue(abdominalMuscle);
+        PhenotypicFeature nemalineRods = PhenotypicFeatureBuilder.create("HP:0003798","Nemaline bodies").build();
+        builder.phenotypicFeature(nemalineRods);
+        ProcedureBuilder pbuilder = ProcedureBuilder.create("NCIT:C51895", "Muscle Biopsy");
+        TimeElement age = TimeElements.age("P1D");
+        pbuilder.bodySite(abdominalMuscle).performed(age);
+        builder.procedure(pbuilder.build());
+        return builder.build();
+    }
+
+    Interpretation interpretation() {
+        InterpretationBuilder ibuilder = InterpretationBuilder.solved("interpretation.id");
+        OntologyClass nm8 = ontologyClass("MONDO:0014138", "nemaline myopathy 8");
+        DiagnosisBuilder dbuilder = DiagnosisBuilder.create(nm8);
+        dbuilder.genomicInterpretation(klhl40InterpretationV1());
+        dbuilder.genomicInterpretation(klhl40InterpretationV2());
+        ibuilder.diagnosis(dbuilder.build());
+        return ibuilder.build();
+    }
+
+
+    /**
+     * ES of umbilical-cord blood revealed two complex
+     * heterozygous mutations to fetal Kelch-like 40 (KLHL40).
+     * Pedigree-based Sanger sequencing confirmed that the mu-
+     * tation c.602G > A (p.Trp201*) was inherited from the
+     * mother (Figure 5) and c.1516A > C (p.Thr506Pro) from
+     * the father (Figure 6)
+     * @return
+     */
+    GenomicInterpretation klhl40InterpretationV1() {
+        // 3: 42686220 (GRCh38) NM_152393.4(KLHL40):c.602G>A (p.Trp201Ter)
+        // ClinVar VCV000060516.7
+        // Canonical SPDI
+        //    NC_000003.12:42686219:G:A
+        // rs397509420
+        //HGNC:30372
+        AlleleBuilder abuilder = AlleleBuilder.create();
+        //abuilder.setSequenceId("ga4gh:VA.GuPzvZoansqNHPoXkQLXKo31VkTpDKsM");
+        abuilder.startEnd( 42686219, 42686220);
+        abuilder.chromosomeLocation("chr3");
+        abuilder.setAltAllele("A");
+        VariationDescriptorBuilder vbuilder = VariationDescriptorBuilder.create("rs397509420");
+        vbuilder.variation(abuilder.buildVariation());
+        vbuilder.genomic();
+        vbuilder.heterozygous();
+        vbuilder.label("NM_152393.4(KLHL40):c.602G>A (p.Trp201Ter)");
+        vbuilder.transcript();
+        GeneDescriptor geneDescriptor = GeneDescriptorBuilder.create("HGNC:30372", "KLHL40").build();
+        vbuilder.geneContext(geneDescriptor);
+        Expression hgvs = Expressions.hgvsCdna("NM_152393.4(KLHL40):c.602G>A");
+        Expression transcriptReference = Expressions.transcriptReference("NM_152393.4");
+        vbuilder.expression(hgvs);
+        vbuilder.expression(transcriptReference);
+        // wrap in VariantInterpretation
+        VariantInterpretationBuilder vibuilder = VariantInterpretationBuilder.create(vbuilder);
+        vibuilder.pathogenic();
+       // vibuilder.actionable();
+
+
+        GenomicInterpretationBuilder gbuilder = GenomicInterpretationBuilder.create("interpretation.1");
```

Comment:
PROBAND_ID

This should be `GenomicInterpretationBuilder.create(PROBAND_ID);` or requires changing to make things clearer. The current output is:

```yaml
phenopacket:
  id: "arbitrary.id"
  subject:
    id: "proband A"
...
interpretations:
  - id: "interpretation.id"
    progressStatus: "SOLVED"
    diagnosis:
      disease:
        id: "MONDO:0014138"
        label: "nemaline myopathy 8"
      genomicInterpretations:
# this should be "proband A"
      - subjectOrBiosampleId: "interpretation.1"

```